### PR TITLE
BZ1854150: Storage workaround for existing clusters

### PR DESCRIPTION
**Pull request by bobfuru**
https://bugzilla.redhat.com/show_bug.cgi?id=1854150
Adding a new module based on this kbase article: https://access.redhat.com/solutions/4952011

Preview link: [Adding container engine storage to installed nodes](https://deploy-preview-36357--osdocs.netlify.app/openshift-enterprise/latest/nodes/nodes/nodes-nodes-resources-configuring?utm_source=github&utm_campaign=bot_dp#nodes-nodes-resources-adding-storage_nodes-nodes-resources-configuring)

### CODE REVIEW

**Comment by bobfuru** [comment]
Hi @control-d - would you mind reviewing this PR for first impressions? I'm interested in your opinion on the placement of the content. I'm also considering using it in the Post-installation --> Machine configuration tasks [doc](https://docs.openshift.com/container-platform/4.8/post_installation_configuration/machine-configuration-tasks.html).

Also, would love your feedback on where I might have missed translating the kbase article. Thanks!

**Comment by bergerhoffer** [comment]
The `branch/enterprise-4.11` label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.10. And any PR going into `main` must also target the latest version branch (enterprise-4.11).

If the update in your PR does NOT apply to version 4.11 onward, please retarget this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

File: modules/nodes-nodes-resources-adding-storage.adoc [hunk 1]
```diff
@@ -0,0 +1,113 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-nodes-resources-configuring.adoc
+
+[id="nodes-nodes-resources-adding-storage_{context}"]
+= Adding container engine storage to installed nodes
```
Comment (sferich888) [inline modules/nodes-nodes-resources-adding-storage.adoc:6]: This section needs to warn uses that by following this on specific parts of the filesystem(fs) that that they are explicitly disabling or inhibiting OpenShift's ability to monitor the file-system usage appropriately see: https://access.redhat.com/articles/4766521

**Comment by bobfuru** [comment]
Hi @control-d - I finally got the updates applied to this PR based on the [gdoc](https://docs.google.com/document/d/1_88IkDRVaAdEoWvGZwjNtnWFrhj8eEM6eTmthATX93k/edit?usp=sharing) you shared with me. Could you PTAL and let me know if you have any suggestions or revisions? Thanks!

**Comment by bergerhoffer** [comment]
The enterprise-4.12 label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.11. And any PR going into `main` must also target the latest version branch (enterprise-4.12).

If the update in your PR does NOT apply to version 4.12 onward, please re-target this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

**Comment by kalexand-rh** [comment]
Only versions 4.8+ are still in maintenance. I am removing labels from earlier versions from this PR.

**Comment by kalexand-rh** [comment]
@control-d will you PTAL? (@lpettyjo, FYI)

**Comment by bergerhoffer** [comment]
The `branch/enterprise-4.13` label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.12. And any PR going into `main` must also target the latest version branch (enterprise-4.13).

If the update in your PR does NOT apply to version 4.13 onward, please re-target this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

**Comment by kalexand-rh** [comment]
@mburke5678 @lpettyjo, are either of you interested in reviving this change?

**Comment by control-d** [comment]
I talked to @jldohmann a few weeks ago about reviving this change. We discussed revisiting it after 4.13 GA.

**Comment by kalexand-rh** [comment]
The `branch/enterprise-4.14` label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.13. And any PR going into `main` must also target the latest version branch (enterprise-4.14).

If the update in your PR does NOT apply to version 4.14 onward, please re-target this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

**Pull request closed without merging**